Pocarski - Combinators

heinwessels opened the pull request:
pocarski

heinwessels committed:
First commit.

heinwessels committed:
Remove artefact.

ClaudeMetz committed:
Editing

DaleStan commented on submissions/pocarski-combinators/pocarski-combinators.md:
```diff
@@ -0,0 +1,47 @@
+
+(counter)
+
+Finally, the counter. Structurally it is identical to the SR latch, but this time the output is set to "input count of everything". This means that, while the decider's condition is followed, it will keep giving its own outputs to itself, thus remembering them. For every tick it receives a signal, it will increment the value of that signal in its memory by the amount received. As soon as the cond ...
```
I think this should say "identical to the RS latch", not SR latch.

DaleStan commented on submissions/pocarski-combinators/pocarski-combinators.md:
```diff
@@ -0,0 +1,47 @@
+
+(backup switch)
+
+Finally, a process that many fear to set up: uranium enrichment. The part we're worried about is the recycling of uranium. We want to stop recycling U-235 as soon as we hit 40 items, after which the extra one should be placed into the output. Naturally, we use a counter for this. The recycling inserter is set to "read hand - pulse", so every time it grabs uranium for recycling the counter counts  ...
```
It may be better when accompanied by an annotated image, but this last sentence seems too long.

pocarski committed:
Update pocarski-combinators.md

Changed "monostable" to "pulser circuit" (the Minecraft redstone community calls circuits that create a pulse from constant signals "monostables")
Changed "SR latch" to "RS latch" and added clarification for "S" and "R" signals
Rewrote the paragraph about enrichment circuit
Changed some wording

heinwessels committed:
Merge pull request #1 from heinwessels/patch-2

Update pocarski-combinators.md

Conor12345 committed:
Tech edit

Conor12345 committed:
counter to pulser switch from
